File: common/src/main/java/me/pandamods/fallingtrees/exceptions/TreeTooBigException.java
package me.pandamods.fallingtrees.exceptions;

import net.minecraft.core.BlockPos;
import net.minecraft.world.level.Level;

public class TreeTooBigException extends TreeException {
	public TreeTooBigException(BlockPos blockPos, Level level) {
		super(String.format("Tree is too big to be processed at position %s in dimension '%s'", blockPos.toShortString(), level.dimension().location()));
	}
}

File: common/src/main/kotlin/exceptions/TreeTooBigException.kt
package dev.pandasystems.fallingtrees.exceptions

import net.minecraft.core.BlockPos
import net.minecraft.world.level.Level

class TreeTooBigException(blockPos: BlockPos, level: Level) :
	TreeException(String.format("Tree is too big to be processed at position %s in dimension '%s'", blockPos.toShortString(), level.dimension().location()))
